### PR TITLE
[NPM] Consolidating default chains and rules for easier readability

### PR DESCRIPTION
**Pull request by vakalapa**
Right now all the basic chain and rules are hard to comprehend with redundant code. Cleaning that up

### CODE REVIEW

File: npm/iptm/helper.go [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+
+	return chainsAndRules
+}
+
+// getAzureNPMChainRules returns all rules for AZURE-NPM chain
+func getAzureNPMChainRules() [][]string {
+	// Note: make sure 0th index is prent chain for logging
+	return [][]string{
+		{
+			util.IptablesAzureChain,
```
Comment (matmerr) [inline npm/iptm/helper.go:33]: let's make a note to potentially wrap these string slices into structs with validation, the potential for ordering mishaps is pretty high

Comment (vakalapa) [reply npm/iptm/helper.go:33]: Noted!

**Review by matmerr — APPROVED** [review]
approved, not sure why the diff shows changes in the pipeline yamls, master should be up to date

**Pull request merged**